src/sql/workbench/contrib/queryplan2/browser/constants.ts

```diff
@@ -0,0 +1,254 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Currently vscode automatically adds vscode-file://vscode-app/<path-to-ads>/out/vs/code/electron-browser/workbench/
+ * to all the icons fetched by the azdatagraph library. This is the relative path to the icons from that dir.
+ */
+let imageBasePath = '../../../../sql/workbench/contrib/queryplan2/browser/images/icons/';
```

Comment:
Why are you doing this instead of resolving the URL relative to this file and then passing the path from that into the lib? Hardcoding paths like this (especially where you're assuming a particular structure of the graph library) is not a great pattern and likely to end up with stuff breaking in the future.